linux_os/guide/services/ssh/ssh_server/sshd_disable_empty_passwords/rule.yml

```diff
@@ -21,6 +21,7 @@ severity: high
 identifiers:
     cce@rhel7: CCE-27471-2
     cce@rhel8: CCE-80896-4
+    cce@sle12: CCE-83014-1
```

Comment:
can you describe where this CCE identifiers are coming from? Do you have a pool of CCE identifiers issued by NIST that you can use for SUSE?

Reply:
Yes, we have a pool of CCE IDs allocated from NIST. Do we need to publish them somewhere? Sorry I am new to this project so not quite familiar with the entire workflow yet. Please let me know.

Reply:
@abergmann can you please clarify the above? Thanks!

Reply:
@guangyee that's exactly the answer I was looking for. From my side I think that should be enough. If others want to ellaborate on need of publishing somewhere then it's out of my knowledge. Thanks

Reply:
@ggbecker Yes, I've got in contact with NIST last year and they gave us a number block for our SLES 12 STIG requirements. :+1:

Reply:
I don't think there is strict need to publish the allocated list of CEE IDs.
But if they are part of the repo, like [shared/references/cce-redhat-avail.txt](https://github.com/ComplianceAsCode/content/blob/master/shared/references/cce-redhat-avail.txt), other people can assign them too.